client-sqldelight-test/src/main/kotlin/app/cash/backfila/client/sqldelight/HockeyPlayerBackfillRecordSourceQueries.kt

```diff
@@ -0,0 +1,41 @@
+package app.cash.backfila.client.sqldelight
+
+import app.cash.backfila.client.sqldelight.hockeydata.HockeyDataDatabase
+import app.cash.backfila.client.sqldelight.hockeydata.HockeyPlayer
+
+fun getHockeyPlayersBackfillQueries(database: HockeyDataDatabase): SqlDelightRecordSourceQueries<Int, HockeyPlayer> = SqlDelightRecordSourceQueries.create(
```

Comment:
this is generated right? does it need to be committed?

Reply:
I probably should have made this a draft for now. What's next for me is using the gradle plugin here, so yes, and expanding the tests before I merge.

Reply:
I am going to do this part in a followup with some help from Jesse.